api/src/foreign_rpc.rs

```diff
@@ -130,116 +130,125 @@ pub trait ForeignRpc: Sync + Send {
 	) -> Result<BlockHeaderPrintable, ErrorKind>;
 
 	/**
-	Networked version of [Foreign::get_block](struct.Foreign.html#method.get_block).
+		Networked version of [Foreign::get_block](struct.Foreign.html#method.get_block).
 
-	# Json rpc example
+		# Json rpc example
 
-	```
-	# grin_api::doctest_helper_json_rpc_foreign_assert_response!(
-	# r#"
-	{
-		"jsonrpc": "2.0",
-		"method": "get_block",
-		"params": [374274, null, null],
-		"id": 1
-	}
-	# "#
-	# ,
-	# r#"
-	{
-		"id": 1,
-		"jsonrpc": "2.0",
-		"result": {
-			"Ok": {
-			"header": {
-				"cuckoo_solution": [
-				1263501,
-				14648727,
-				42430559,
-				58137254,
-				68666726,
-				72784903,
-				101936839,
-				104273571,
-				123886748,
-				131179768,
-				155443226,
-				162493783,
-				164784425,
-				167313215,
-				169806918,
-				183041591,
-				184403611,
-				210351649,
-				215159650,
-				239995384,
-				240935454,
-				257742462,
-				280820644,
-				300143903,
-				303146496,
-				311804841,
-				341039986,
-				354918290,
-				363508555,
-				377618528,
-				396693709,
-				397417856,
-				399875872,
-				413238540,
-				413767813,
-				432697194,
-				436903767,
-				447257325,
-				453337210,
-				459401597,
-				496068509,
-				511300624
+
+	curl -X POST \
+		 -H 'Content-Type: application/json' \
+		 -d '{"jsonrpc":"2.0","id":1,"method":"get_block","params":[952019, null, null]}' \
+		 http://localhost:3413/v2/foreign
+
```

Comment:
What's that? The doctests (currently disabled) are just below.

Reply:
lol 😭